static/app/views/explore/content.tsx

```diff
@@ -104,3 +116,106 @@ export function ExploreContent() {
     </SentryDocumentTitle>
   );
 }
+
+interface SpansTabTourProps {
+  children: ReactNode;
+}
+
+function SpansTabTour({children}: SpansTabTourProps) {
+  const {data: assistantData} = useAssistant();
+  const isTourCompleted = useMemo(() => {
+    const tourData = assistantData?.find(
+      item => item.guide === EXPLORE_SPANS_TOUR_GUIDE_KEY
+    );
+
+    // Prevent tour from showing until assistant data is loaded
+    return tourData?.seen ?? true;
+  }, [assistantData]);
```

Comment:
This might also be worth extracting out into something reusable, again not for this PR but calling it out for later on